### DISCUSSION TITLE
Is it safe when I keep SIP disabled?

### DISCUSSION THREAD

**Discussion by noridaneee** [chris1111/Wireless-USB-Big-Sur-Adapter#67] [Q&A]
Hi! I was just wondering if my device's safety from malware will be compromised if I keep the SIP disabled? Or how should I keep my mac from having malware when I keep the SIP disabled. 

Cheers

**Comment by chris1111** [comment]
See this https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/discussions/52
 This https://eclecticlight.co/2019/06/19/what-is-sip-and-when-is-it-safe-to-turn-it-off/

**Comment by chris1111** [comment]
I have myself the SIP Disable since OS X El Capitan (2015) I never have a Malware or any other shit ... 👾 🤖 🎃👽 ☠️
If I doubt about some file I have scan [with Virustotal](https://www.virustotal.com/gui/)

😂 ➤ [Scan file](https://www.virustotal.com/gui/file/87df2a3469cdcda4ee3b26f9f9d3f3c396586c0b751c7473c127cd774f2d68c8/detection)

**Reply by noridaneee** [reply]
Alright. Thanks Chris!
